flink-connectors/flink-sql-connector-hive-3.1.3/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,52 @@
+flink-sql-connector-hive-3.1.3
+Copyright 2014-2022 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.avro:avro:1.8.2
+- org.apache.avro:avro-mapred:hadoop2:1.8.2
+- org.apache.hive:hive-exec:3.1.3
+
+This project bundles the following dependencies under the BSD license.
+See bundled license files for details.
+
+- org.antlr:antlr-runtime:3.5.2
+
+The bundled Apache Hive org.apache.hive:hive-exec dependency bundles the following dependencies under
+the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
```

Comment:
We still do not have Guava in the NOTICE. Shall we jump on a call tomorrow and discuss? I feel like we are not making progress. I will slack you.

Reply:
Thank you @dannycranmer for helping me out in understanding the `NOTICE` related things specific to flink